Makefile

```diff
@@ -1,11 +1,11 @@
+test: test_php test_python
+
 test_php:
-	@echo "\n\033[92m\033[1mPHP\033[0m\033[0m"
-	./vendor/bin/phpunit --colors --verbose --bootstrap ./tests/php/phpunit.php ./tests/php/
+	@echo -e "\n\033[92m\033[1mPHP\033[0m\033[0m"
```

Comment:
Adding the `-e` flag seems to cause it to be outputted as well when running the tests. It might be better to switch from `echo` to `printf` (and add a linebreak `\n` in the end of the string).

Reply:
Replaced with printf to make the output more coherent in different system.